end-to-end/0-serial/011-websocket/ambassador/service.yaml

```diff
@@ -0,0 +1,15 @@
+aws:
```

Comment:
Huh. This'll always work in Kubernaut? Hey @plombardi89, is adding an AWS dependency here a good idea right now?

Reply:
Wait, this file should not even be here. This is auto generated IIRC. Removing.

Reply:
Removed